packages/data-clients/core/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@terra-ui-packages/core",
```

Comment:
This should be named something related to `data-client`... `core` is too generic.

```suggestion
  "name": "@terra-ui-packages/data-client-core",
```

Reply:
ah yeah, the directory structure differing from the package name threw me off here

Reply:
We _could_ continue that convention and have `packages/data-clients/data-client-core`, etc...